### DISCUSSION TITLE
Use iOS device instead of Android

### DISCUSSION THREAD

**Discussion by aslios** [sbertix/Swiftagram#113] [Q&A] [answered]
Hi Sbertix,
I want to change login device name? I want use iphone or other apple device name instead of android and when i login via SwiftGram instagram send to me mail as "some android device (samsung xxxx ) login etc. ..."
i dont want see android device name.can i write "iphone" + "device name" ? (like "aslios iphone" )
Thank you so much

**Comment by sbertix** [comment, marked as answer]
Hey @aslios,

There are many issues closed already regarding this feature. 
All you need to do is set the following line on app launch.

```swift
Client.default = .current ?? .iPhone11ProMax
```

This will try and detect the user's iPhone. But if it's not an iPhone, it's gonna use the iPhone 11 Max as reference.

**Comment by sbertix** [comment]
I just wanted to add one thing: starting with the next "bigger" version (`4.3.0`, if there's gonna be one, or `5.0.0`) `.current ?? .iPhone11ProMax` is going to be the default `Client`. 

I feel like it's been tested enough and it's expected behavior.

**Reply by aslios** [reply]
thank you I applied your solution its good but mail came like this,
Apple iphone ,Mobile SAFARI - London
          England
How can i delete this "Mobile SAFARI" text ?

**Reply by sbertix** [reply]
Obviously 😊 

You're using a Safari web view to log in. It will always say Safari.
That's expected behavior. 
Nothing I can do unfortunately. Apologies
